### PR TITLE
MAINTAINERS.md: add Rudloff and Proscream

### LINKED ISSUE #3580: add collaborator: Rudloff
Hi, @Rudloff! You seem to be enjoying contributing to the tldr-pages project.
You now have had five distinct pull requests merged (#3566, #3541, #3545, #3544, #3552)!
That qualifies you to become a collaborator in this repository, as explained in our [community roles documentation](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md).

As a collaborator, you will have commit access to the repository.
That means you can merge pull requests, label and close issues, and perform various other maintenance tasks that are needed here and there.
Of course, all of this is voluntary — you're welcome to contribute to the project in whatever ways suit your liking.

If you do decide to start performing maintenance tasks, though, we only ask you to get familiar with the [maintainer's guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md).

So, what do you say? Can we add you as a collaborator?

Either way, thanks for all your work so far!

### LINKED ISSUE #3592: add contributor: @Proscream
Hi, @Proscream! You seem to be enjoying contributing to the tldr-pages project.
You now have had five distinct pull requests merged (#3549, #3553, #3560, #3577, #3578, #3579, #3582, #3585, and #3586)!
That qualifies you to become a collaborator in this repository, as explained in our [community roles documentation](https://github.com/tldr-pages/tldr/blob/master/COMMUNITY-ROLES.md).

As a collaborator, you will have commit access to the repository.
That means you can merge pull requests, label and close issues, and perform various other maintenance tasks that are needed here and there.
Of course, all of this is voluntary — you're welcome to contribute to the project in whatever ways suit your liking.

If you do decide to start performing maintenance tasks, though, we only ask you to get familiar with the [maintainer's guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/maintainers-guide.md).

So, what do you say? Can we add you as a collaborator?

Either way, thanks for all your work so far!

### PR DESCRIPTION
**Pull request by sbrl**
Oops.... I should have done this before. Thanks to @waldyrious for reminding me!

Closes #3580
Closes #3592

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ If you are an owner of the organization, you should be able to see an automated
   [9 May 2019](https://github.com/tldr-pages/tldr/issues/2988) - present
 - **Ein Verne ([@einverne](https://github.com/einverne))**:
   [29 October 2019](https://github.com/tldr-pages/tldr/issues/3488) - present
+- **Proscream ([@Proscream](https://github.com/Proscream))**:
+[22nd November 2019](https://github.com/tldr-pages/tldr/issues/3592) - present
+- **Pierre Rudloff ([@Rudloff](https://github.com/Rudloff))**:
+[19th November 2019](https://github.com/tldr-pages/tldr/issues/3592) - present
```
Comment (waldyrious) [inline MAINTAINERS.md]: We're not using ordinal format for the dates :) please change these to match the existing ones.

Comment (waldyrious) [reply MAINTAINERS.md]: Also, the links are duplicated; and finally, the entries should be in chronological order.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -35,10 +35,10 @@ If you are an owner of the organization, you should be able to see an automated
   [9 May 2019](https://github.com/tldr-pages/tldr/issues/2988) - present
 - **Ein Verne ([@einverne](https://github.com/einverne))**:
   [29 October 2019](https://github.com/tldr-pages/tldr/issues/3488) - present
-- **Proscream ([@Proscream](https://github.com/Proscream))**:
-[22nd November 2019](https://github.com/tldr-pages/tldr/issues/3592) - present
 - **Pierre Rudloff ([@Rudloff](https://github.com/Rudloff))**:
-[19th November 2019](https://github.com/tldr-pages/tldr/issues/3592) - present
+  [19 November 2019](https://github.com/tldr-pages/tldr/issues/3580) present
```
Comment (waldyrious) [inline MAINTAINERS.md]: Missing the dash before "present" :)

**Comment by sbrl** [comment]
Fixed, @waldyrious - thanks :-)

File: MAINTAINERS.md (hunk 1 above)
Comment (bl-ue) [reply MAINTAINERS.md]: Haha - looks like this old conversation was never resolved.  Resolving it now :)

**Pull request merged**